src/tox/package.py

```diff
@@ -0,0 +1,85 @@
+import sys
+
+import py
+
+import tox
+
+
+@tox.hookimpl
+def tox_package(session, venv):
+    """Build an sdist at first call return that for all calls"""
+    if not hasattr(session, "package"):
+        session.package = get_package(session)
+    return session.package
+
+
+def get_package(session):
+    """"Perform the package operation"""
+    config, report = session.config, session.report
+    if config.skipsdist:
+        report.info("skipping sdist step")
+        return None
+    if not config.option.sdistonly and (config.sdistsrc or config.option.installpkg):
+        path = config.option.installpkg
+        if not path:
+            path = config.sdistsrc
+        path = session._resolve_package(path)
+        report.info("using package {!r}, skipping 'sdist' activity ".format(str(path)))
+    else:
+        try:
+            path = make_sdist(report, config, session)
+        except tox.exception.InvocationError:
+            v = sys.exc_info()[1]
```

Comment:
(I know this is existing, but I couldn't help notice it) -- shouldn' tthis be `as v:` and not retrieving `sys.exc_info()`?

Reply:
Indeed it should :+1: